README.md

```diff
@@ -206,7 +206,7 @@ Enable the Autopause functionality by setting:
 -e ENABLE_AUTOPAUSE=TRUE
 ```
 
-There are 4 more environment variables that define the behaviour:
+There are 5 more environment variables that define the behaviour:
```

Comment:
That's when I change my wording to be like
```suggestion
The following environment variables define the behaviour of auto-pausing:
```

Reply:
Fair enough, done